tests/unit/mixins/fastboot-compat-test.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+import FastbootCompatMixin from 'ember-user-activity/mixins/fastboot-compat';
+import { module, test } from 'qunit';
+
+module('Unit | Mixin | fastboot compat');
+
+// Replace this with your real tests.
```

Comment:
Can you add a test here that just registers a mock `fastboot` service, and another one that doesn't (and confirms that `isFastBoot` is false as a result)>